### PR TITLE
Support checkmode in pointcloud segmentation and fix count problems

### PR DESCRIPTION
**Pull request by MMMMoriaty**

### CODE REVIEW

File: packages/lb-annotation/src/core/pointCloud/segmentation.ts [hunk 1]
```diff
@@ -32,6 +35,7 @@ class PointCloudSegmentOperation {
   constructor(props: IProps) {
     this.dom = props.dom;
     this.store = props.store;
+    this.checkMode = props.checkMode;
```
Comment (Kerwin-L) [inline packages/lb-annotation/src/core/pointCloud/segmentation.ts:38]: Don't need to pass  checkMode to segmentation directly. Saving checkMode to the store is a better way.

Comment (MMMMoriaty) [reply packages/lb-annotation/src/core/pointCloud/segmentation.ts:38]: done

Comment (MMMMoriaty) [reply packages/lb-annotation/src/core/pointCloud/segmentation.ts:38]: but i see 'isForbid' in segmentation, don't you think 'isForbid' is the same as 'checkMode' in some way?

Comment (Kerwin-L) [reply packages/lb-annotation/src/core/pointCloud/segmentation.ts:38]: Yeah, but `isForbid` use `store.forbidOperation`. You can add multiple conditions for evaluation like
```ts
  public get isForbid() {
    return this.forbidOperation || this.store.checkMode;
  }
```

**Pull request merged**